testsuite/features/srv_content_lifecycle.feature

```diff
@@ -0,0 +1,142 @@
+# Copyright (c) 2019 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Content lifecycle management 
+
+  Scenario: I create a content lifecycle project Given I am authorized as "admin" with password "admin"
+    Given I am authorized as "admin" with password "admin"
+    Then I follow "Content Lifecycle Management"
+    Then I follow "Content Lifecycle Projects"
+    And I should see a "Content Lifecycle Projects" text
+    And I should see a "There are no entries to show." text
+    Then I follow "Create Project" 
+    And I should see a "Create a new Content Lifecycle Project" text
+    And I should see a "Project Properties" text
+    Then I enter "clp_label" as "label" 
+    Then I enter "clp_name" as "name" 
+    Then I enter "clp_desc" as "description" 
```

Comment:
1. And

2. Could we use some meaningful English description instead of "clp_desc"?